packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -0,0 +1,271 @@
+import { existsSync, mkdirSync } from "fs";
+import { join } from "path";
+import alias from "@rollup/plugin-alias";
+import { getBabelInputPlugin, getBabelOutputPlugin } from "@rollup/plugin-babel";
+import commonjs from "@rollup/plugin-commonjs";
+import json from "@rollup/plugin-json";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import typescript from "@rollup/plugin-typescript";
+import loadConfigFile from "rollup/dist/loadConfigFile";
+import clear from "rollup-plugin-clear";
+import command from "rollup-plugin-command";
+import livereload from "rollup-plugin-livereload";
+import sass from "rollup-plugin-sass";
+import { terser } from "rollup-plugin-terser";
+import { cp } from "shelljs";
+import { zip } from "zip-a-folder";
+import { widgetTyping } from "./rollup-plugin-widget-typing";
+import {
+    editorConfigEntry,
+    isTypescript,
+    previewEntry,
+    projectPath,
+    sourcePath,
+    widgetEntry,
+    widgetName,
+    widgetPackage,
+    widgetVersion
+} from "./shared";
+
+const outDir = join(sourcePath, "/dist/tmp/widgets/");
+const outWidgetFile = join(widgetPackage.replace(/\./g, "/"), widgetName.toLowerCase(), `${widgetName}.js`);
+const mpkDir = join(sourcePath, "dist", widgetVersion);
+const mpkFile = join(mpkDir, `${widgetPackage}.${widgetName}.mpk`);
+
+// Notes for future me:
+// - plugins that transform non-js code to js should go before plugins that work with js
+// - for editorConfig we target JS engine in Studio Pro, which supports only es5 and no source maps
```

Comment:
i think first comment really doesn't add much and second one is better closer to the source code